source/reference/modules/studio/configuration/index.rst

```diff
@@ -462,9 +462,26 @@ The X-PERMITTED-CROSS-DOMAIN-POLICIES header is set to ``none`` (do not allow an
     # Value for the X-PERMITTED-CROSS-DOMAIN-POLICIES header
     studio.security.headers.permittedCrossDomainPolicies.value: none
 
-
 |
 
+"""""""""""""""
+Referrer Policy
+"""""""""""""""
+.. version_tag::
+    :label: Since
+    :version: 4.3.1
+
+The following allows you to limit the information available in the Referer header or to not send the Referer header.
+The referrer policy header is set to ``NO_REFERRER`` (Never send the Referer header) by default.
```

Comment:
Add a note: `Possible values are defined in org.springframework.security.web.header.writers.ReferrerPolicyHeaderWriter.ReferrerPolicy`